File: Cplusplus/나누어 떨어지는 숫자 배열.cpp
//22.01.17 나누어 떨어지는 숫자 배열 by CJHofficial
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr, int divisor) {
    vector<int> answer;
    for (int num : arr) {
        if ((num % divisor) == 0) {
            answer.push_back(num);
        }
    }
    sort(answer.begin(), answer.end());
    if (answer.size() == 0) answer.push_back(-1);
    return answer;
}

File: Python/level1/나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    answer = [number for number in sorted(arr) if number % divisor == 0]
    return answer if len(answer) !=0 else [-1]

# return A or B => or연산자를 통해 A == False일 경우 B를 내보낼 수 있다.  
